py_gtfs_rt_ingestion/lib/s3_utils.py

```diff
@@ -46,25 +45,13 @@ def file_list_from_s3(bucket_name: str, file_prefix: str) -> List[str]:
     :param bucket_name: the name of the bucket to look inside of
     :param file_prefix: prefix for files to generate
 
-    :return list of s3 filepaths sorted by the timestamps formatted into them
+    :return list of s3 filepaths
     """
     process_logger = ProcessLogger(
         "file_list_from_s3", bucket_name=bucket_name, file_prefix=file_prefix
     )
     process_logger.log_start()
 
-    def strip_timestamp(fileobject: str) -> str:
```

Comment:
rather than doing the sort here, do it in the utility that buckets these by filetype.